CMakeLists.txt

```diff
@@ -78,11 +78,8 @@ tristate_option(WITH_USDT
   "if sys/sdt.h is found."
   AUTO
 )
-tristate_option(WITH_EXTERNAL_SIGNER
-  "Enable external signer support."
-  "if Boost.Process is found."
-  AUTO
-)
+cmake_dependent_option(WITH_EXTERNAL_SIGNER "Enable external signer support." ON "NOT WIN32" OFF)
+set(ENABLE_EXTERNAL_SIGNER ${WITH_EXTERNAL_SIGNER})
```

Comment:
Why this `set`? Is it just because the option name and the internal name are different?

Reply:
Correct. See: https://github.com/hebasto/bitcoin/blob/b8fd28543e19f355e494f27bffc1e0c1ebcb54ce/cmake/bitcoin-config.h.in#L145-L146

Reply:
Thanks!